From a4f4031323aea5e8651921b4a6308c9f93df3e64 Mon Sep 17 00:00:00 2001
From: Bhaskar Upadhaya <Bhaskar.Upadhaya@nxp.com>
Date: Fri, 9 Dec 2016 07:28:36 +0530
Subject: [PATCH 301/508] LS1012:its Add its for LS1012A Platform

Below its are added
kernel-ls1012a-rdb.its
kernel-ls1012a-qds.its
kernel-ls1012a-frdm.its

Signed-off-by: Bhaskar Upadhaya <Bhaskar.Upadhaya@nxp.com>
[Original patch taken from SDK-V2.0-1703]
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 kernel-ls1012a-frdm.its |   53 +++++++++++++++++++++++++++++++++++++++++++++++
 kernel-ls1012a-qds.its  |   53 +++++++++++++++++++++++++++++++++++++++++++++++
 kernel-ls1012a-rdb.its  |   53 +++++++++++++++++++++++++++++++++++++++++++++++
 3 files changed, 159 insertions(+), 0 deletions(-)
 create mode 100644 kernel-ls1012a-frdm.its
 create mode 100644 kernel-ls1012a-qds.its
 create mode 100644 kernel-ls1012a-rdb.its

diff --git a/kernel-ls1012a-frdm.its b/kernel-ls1012a-frdm.its
new file mode 100644
index 0000000..c223d8c
--- /dev/null
+++ b/kernel-ls1012a-frdm.its
@@ -0,0 +1,53 @@
+/*
+ * Copyright (C) 2016, Freescale Semiconductor
+ *
+ * This file is licensed under the terms of the GNU General Public
+ * License version 2.  This program is licensed "as is" without any
+ * warranty of any kind, whether express or implied.
+ */
+
+/dts-v1/;
+
+/ {
+	description = "Image file for the LS1012A Linux Kernel";
+	#address-cells = <1>;
+
+	images {
+		kernel@1 {
+			description = "ARM64 Linux kernel";
+			data = /incbin/("./arch/arm64/boot/Image");
+			type = "kernel";
+			arch = "arm64";
+			os = "linux";
+			compression = "none";
+			load = <0x80080000>;
+			entry = <0x80080000>;
+		};
+		fdt@1 {
+			description = "Flattened Device Tree blob";
+			data = /incbin/("./arch/arm64/boot/dts/freescale/fsl-ls1012a-frdm.dtb");
+			type = "flat_dt";
+			arch = "arm64";
+			compression = "none";
+			load = <0x90000000>;
+		};
+		ramdisk@1 {
+			description = "LS2 Ramdisk";
+                        data = /incbin/("./fsl-image-core-ls1012afrdm-20160616014706.rootfs.ext2.gz");
+			type = "ramdisk";
+			arch = "arm64";
+			os = "linux";
+			compression = "none";
+		};
+	};
+
+	configurations {
+		default = "config@1";
+		config@1 {
+			description = "Boot Linux kernel";
+			kernel = "kernel@1";
+			fdt = "fdt@1";
+			ramdisk = "ramdisk@1";
+		};
+	};
+};
diff --git a/kernel-ls1012a-qds.its b/kernel-ls1012a-qds.its
new file mode 100644
index 0000000..4e3da41
--- /dev/null
+++ b/kernel-ls1012a-qds.its
@@ -0,0 +1,53 @@
+/*
+ * Copyright (C) 2016, Freescale Semiconductor
+ *
+ * This file is licensed under the terms of the GNU General Public
+ * License version 2.  This program is licensed "as is" without any
+ * warranty of any kind, whether express or implied.
+ */
+
+/dts-v1/;
+
+/ {
+	description = "Image file for the LS1012A Linux Kernel";
+	#address-cells = <1>;
+
+	images {
+		kernel@1 {
+			description = "ARM64 Linux kernel";
+			data = /incbin/("./arch/arm64/boot/Image");
+			type = "kernel";
+			arch = "arm64";
+			os = "linux";
+			compression = "none";
+			load = <0x80080000>;
+			entry = <0x80080000>;
+		};
+		fdt@1 {
+			description = "Flattened Device Tree blob";
+			data = /incbin/("./arch/arm64/boot/dts/freescale/fsl-ls1012a-qds.dtb");
+			type = "flat_dt";
+			arch = "arm64";
+			compression = "none";
+			load = <0x90000000>;
+		};
+		ramdisk@1 {
+			description = "LS2 Ramdisk";
+                        data = /incbin/("./fsl-image-core-ls1012ardb-20160428083154.rootfs.ext2.gz");
+			type = "ramdisk";
+			arch = "arm64";
+			os = "linux";
+			compression = "none";
+		};
+	};
+
+	configurations {
+		default = "config@1";
+		config@1 {
+			description = "Boot Linux kernel";
+			kernel = "kernel@1";
+			fdt = "fdt@1";
+			ramdisk = "ramdisk@1";
+		};
+	};
+};
diff --git a/kernel-ls1012a-rdb.its b/kernel-ls1012a-rdb.its
new file mode 100644
index 0000000..dc74ade
--- /dev/null
+++ b/kernel-ls1012a-rdb.its
@@ -0,0 +1,53 @@
+/*
+ * Copyright (C) 2016, Freescale Semiconductor
+ *
+ * This file is licensed under the terms of the GNU General Public
+ * License version 2.  This program is licensed "as is" without any
+ * warranty of any kind, whether express or implied.
+ */
+
+/dts-v1/;
+
+/ {
+	description = "Image file for the LS1012A Linux Kernel";
+	#address-cells = <1>;
+
+	images {
+		kernel@1 {
+			description = "ARM64 Linux kernel";
+			data = /incbin/("./arch/arm64/boot/Image");
+			type = "kernel";
+			arch = "arm64";
+			os = "linux";
+			compression = "none";
+			load = <0x80080000>;
+			entry = <0x80080000>;
+		};
+		fdt@1 {
+			description = "Flattened Device Tree blob";
+			data = /incbin/("./arch/arm64/boot/dts/freescale/fsl-ls1012a-rdb.dtb");
+			type = "flat_dt";
+			arch = "arm64";
+			compression = "none";
+			load = <0x90000000>;
+		};
+		ramdisk@1 {
+			description = "LS2 Ramdisk";
+                        data = /incbin/("./fsl-image-core-ls1012ardb-20160616014215.rootfs.ext2.gz");
+			type = "ramdisk";
+			arch = "arm64";
+			os = "linux";
+			compression = "none";
+		};
+	};
+
+	configurations {
+		default = "config@1";
+		config@1 {
+			description = "Boot Linux kernel";
+			kernel = "kernel@1";
+			fdt = "fdt@1";
+			ramdisk = "ramdisk@1";
+		};
+	};
+};
-- 
1.7.5.4

